### ISSUE TITLE
Provide a release tarball

### ISSUE THREAD

**Issue by fcambus** [saitoha/libsixel#64]
It would be nice if you would consider adding proper source release tarballs,
as GitHub auto-generated tarballs are not immutable. This is an issue [1] for
operating systems creating packages for libsixel. Thanks in advance.

[1] https://marc.info/?l=openbsd-ports&m=151973450514279&w=2

**Comment by hpjansson** [comment]
`make distcheck` also packages up-to-date *configure*, *Makefile.in* etc. files so you don't have to keep them in the repo, and it tries to make sure your tarball can be built and installed without errors.

**Comment by saitoha** [comment]
@fcambus Uploaded. Is it OK?
https://github.com/saitoha/libsixel/releases/download/v1.7.3/libsixel-1.7.3.tar.gz

@hpjansson Thanks for your advice, but I still wonder whether `configure` should be removed from repo. because it force all contributers to install autotools/libtool. (I think almost of them just only want to do trivial fixes...)

**Comment by hpjansson** [comment]
@saitoha It's generally considered best practice for a couple of reasons laid out here: https://stackoverflow.com/a/3291181

I just mentioned it since I thought you'd consider it a bonus, though. Not trying to hijack the issue :slightly_smiling_face:

**Comment by fcambus** [comment]
@saitoha Yes, that's perfect, thanks a lot! I switched the OpenBSD port to fetch the newly uploaded release.
